spec/models/user_spec.rb

```diff
@@ -1,5 +1,20 @@
 require 'rails_helper'
 
 RSpec.describe User, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "#reviews_by_airline" do
+    context 'user has reviews' do
+      user1 = FactoryGirl.create(:review).user
+
+      it 'should return reviews namespaced by airline' do
+        expect(user1.reviews_by_airline).to eq({user1.reviews.first.airline => [user1.reviews.first]})
```

Comment:
Line is too long. [102/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.